blockchain/api.go

```diff
@@ -147,6 +152,9 @@ var DealOpenedTopic common.Hash = common.HexToHash("0x873cb35202fef184c9f8ee23c0
 var DealAcceptedTopic common.Hash = common.HexToHash("0x3a38edea6028913403c74ce8433c90eca94f4ca074d318d8cb77be5290ba4f15")
 var DealClosedTopic common.Hash = common.HexToHash("0x72615f99a62a6cc2f8452d5c0c9cbc5683995297e1d988f09bb1471d4eefb890")
 
+var ErrorLogNotFound = errors.New("Log not found")
```

Comment:
Just a "log"? I think it's too general, so just add the "transaction" word to an error text :)